### PR TITLE
[CDAP-13118] Modify EntityId usage from UI

### PR DESCRIPTION
**Pull request by elfenheart**
Corresponding Backend Change: https://github.com/caskdata/cdap/pull/9838

JIRA: https://issues.cask.co/browse/CDAP-13118

### CODE REVIEW

**Review by tonybach — COMMENTED** [review]
Code LGTM 👍

File: cdap-ui/app/cdap/components/AppDetailedView/Tabs/HistoryTab.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default class HistoryTab extends Component {
       });
   }
 
-  renderTableBody(history) {
+  renderTableBody = (history) => {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/AppDetailedView/Tabs/HistoryTab.js:119]: nit: If we are touching this file can we remove the constructor and have things as class properties (like `ProgramTable/index.js`)

**Review by ajainarayanan — COMMENTED** [review]
@elfenheart I am still seeing references to `datasetinstance` in some fast actions and dataset detailed view etc., We don't want to change those references?

**Comment by ajainarayanan** [comment]
👍 LGTM

**Pull request merged**